exporter/clickhouseexporter/factory.go

```diff
@@ -101,3 +103,26 @@ func createTracesExporter(
 		exporterhelper.WithRetry(c.RetrySettings),
 	)
 }
+
+func createMetricExporter(
+	ctx context.Context,
+	set component.ExporterCreateSettings,
+	cfg component.ExporterConfig,
```

Comment:
use `cfg component.Config`  instead, you can following details in https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/a8fb43b24d4f2ae93ef008f4ca8c9f4a0630d90b/exporter/clickhouseexporter/factory.go#L60